x-pack/plugins/ml/public/application/data_frame_analytics/common/analytics.ts

```diff
@@ -106,23 +107,12 @@ export enum INDEX_STATUS {
   ERROR,
 }
 
-export interface FieldSelectionItem {
-  name: string;
-  mappings_types?: string[];
-  is_included: boolean;
-  is_required: boolean;
-  feature_type?: string;
-  reason?: string;
-}
-
-export interface DfAnalyticsExplainResponse {
-  field_selection?: FieldSelectionItem[];
-  memory_estimation: {
-    expected_memory_without_disk: string;
-    expected_memory_with_disk: string;
-  };
+export interface FieldSelectionItem
```

Comment:
these types might be better placed in the types folder

Reply:
Moved those to the common folder in f62dad344f36fd8fcd405c98394d91606b37be08
Will move the rest in the follow up - along with some reorg.

Reply:
Moved all relevant types to common dir in 8539c8b539e4580589f8100b9397cedf18d92659